packages/client/src/orders/types/getGuestOrders.types.ts

```diff
@@ -0,0 +1,8 @@
+import type { OrderSummary } from './orderSummary.types';
+
+export type GetGuestOrders = (
+  guestUserEmail: string,
+  config?: Record<string, unknown>,
+) => Promise<OrderSummary>;
```

Comment:
This should return an array of `OrderSummary` I believe. Can you confirm it?

Reply:
You're correct, except it is an `Order[]` instead of `OrderSummary[]`.
Fixed it.